actioncable/app/javascript/actioncable/helpers.js

```diff
@@ -0,0 +1,16 @@
+
+export function startDebugging() {
+  return this.debugging = true
+}
+
+export function stopDebugging() {
+  return this.debugging = null
+}
```

Comment:
The `debugging` flag should only be set on the top level object, but these will set it on whatever happens to be `this`. I'd avoid using `this` in helpers, generally.